src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
+        int salarySecond = 0;
+        int salaryThird = 0;
+        String firstName = "";
+        String secondName = "";
+        String thirdName = "";
+
+        String [] splitDataFrom = dateFrom.split("[.]");
+        String [] splitDataTo = dateTo.split("[.]");
+        String [] splitCurrentData = new String[3];
+        String [] splitInputString = new String[4];
+
+        int currentDay = 0;
```

Comment:
Use LocalDate instead of Date. You should use the LocalDate class from the java.time package instead of working with integers for days and months.